tests/cases/fourslash/aliasMergingWithNamespace.ts

```diff
@@ -0,0 +1,7 @@
+///<reference path="fourslash.ts"/>
+
+////namespace bar { }
+////import bar = bar/**/;
+
+goTo.marker();
+verify.quickInfoExists();
```

Comment:
Ew.  We have a "quickInfoExists"?  can we actually ask for the actual quick info?  I mean, i'm worried htat quick info one time will fail, return like an empty string for some reason, and this mehtod will think things are ok.  just use verify.quickInfoIs("...").